datasets/civieleconstructies/dataset.json

```diff
@@ -7,7 +7,7 @@
       "infra en civiel"
     ],
     "homepage": "https://data.amsterdam.nl",
-    "owner": "Directie stadswerken, Gemeente Amsterdam",
```

Comment:
hoofdletter voor Stadwerken?

Reply:
owner is eigenlijk naar "Directie Verkeer en Openbare Ruimte, Gemeente Amsterdam" gewijzigd. Geen idee hoe  je aan "Directie stadswerken, Gemeente Amsterdam komt"? 

![afbeelding](https://github.com/Amsterdam/amsterdam-schema/assets/25054985/95461607-0a51-4af9-b697-ae17bff423f3)